aptabase/src/main/java/com/aptabase/Aptabase.kt

```diff
@@ -43,7 +43,8 @@ class Aptabase private constructor() {
       "SH" to ""
     )
 
-  fun initialize(context: Context, appKey: String, opts: InitOptions? = null) {
+  fun initialize(context: Context, appKey: String, opts: InitOptions? = null, trackingEnabled: Boolean = true) {
```

Comment:
Can we add the `trackingEnabled` option to `InitOptions`?
Also, why are the units deleted?

Reply:
Would be an option to put them there...
Unit tests are not necessary and not written so why keeping it?

Reply:
Thinking more about this, I don't really find it valuable to add this option to the sdk. You can just wrap the TrackEvent method and handle it yourself as a consumer of the sdk